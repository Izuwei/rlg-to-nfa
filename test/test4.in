A,B
a,b,c
A
A->abcabcabcB
B->bb
