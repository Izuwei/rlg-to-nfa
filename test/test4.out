A,A1,A2,A3,A4,A5,A6,A7,A8,B,B1,B2
a,b,c
A
A->aA1
A1->bA2
A2->cA3
A3->aA4
A4->bA5
A5->cA6
A6->aA7
A7->bA8
A8->cB
B->bB1
B1->bB2
B2->#
